setup.py

```diff
@@ -5,7 +5,7 @@
 with open("requirements.txt") as requirements:
     requirements = requirements.readlines()
 
-with open("README.rst") as f:
+with open("README.md") as f:
```

Comment:
There's still settings in `setup` that need to be updated, like `author_email` and `url`.